dev-python/jsonschema/jsonschema-4.4.0-r2.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
Why are you adding two new revisions?

Reply:
both versions existed when I first submitted the PR. Solved now.